packages/twilio-run/src/checks/nodejs-version.ts

```diff
@@ -22,7 +22,11 @@ export function printVersionWarning(
 
 export default function checkNodejsVersion() {
```

Comment:
@dkundel Is there a way, I can these this piece of code? Looks like this gets called only during `twilio serverless:start`. But I am not sure how I can pass various node runtimes to test this logic.

Reply:
Verified using the following commands
- Change node version using `nvm use` command
- Start the project using `twilio serverless:start`

Updated the test results to the PR description.